package.json

```diff
@@ -4,9 +4,12 @@
   "description": "Auto-tabulation for your inputs. Tiny helper that focus on the next field when a field reach maxLength.",
   "main": "index.js",
   "dependencies": {},
-  "devDependencies": {},
+  "devDependencies": {
+    "standard": "^12.0.1"
+  },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "standard ."
```

Comment:
I guess we could add `--fix` here, to automatically fix what can be fixed.

Reply:
Done 🎉